fuel-txpool/src/service.rs

```diff
@@ -1,143 +1,108 @@
-use std::cmp::Reverse;
+use crate::{interface::Interface, Config};
+use fuel_core_interfaces::block_importer::ImportBlockBroadcast;
+use fuel_core_interfaces::txpool::{TxPoolDb, TxPoolMpsc, TxStatusBroadcast};
 use std::sync::Arc;
-
-use crate::Error;
-use crate::{subscribers::MultiSubscriber, types::*, Config, TxPool as TxPoolImpl};
-use async_trait::async_trait;
-use fuel_core_interfaces::model::{ArcTx, TxInfo};
-use fuel_core_interfaces::txpool::{Subscriber, TxPool, TxPoolDb};
-use std::collections::HashMap;
-use tokio::sync::RwLock;
-
-pub struct TxPoolService {
-    txpool: RwLock<TxPoolImpl>,
-    db: Box<dyn TxPoolDb>,
-    subs: MultiSubscriber,
+use tokio::sync::{broadcast, mpsc, Mutex};
+use tokio::task::JoinHandle;
+use tracing::warn;
+
+pub struct Service {
```

Comment:
Is there a reason the name `TxPoolService` was changed? I could see `Service` leading to conflicts in the future as new parts are introduced, something more distinct could avoid needing alias it on imports

Reply:
All modules have the same naming, all of them now have `Service` and `Config` structures inside